tsconfig.json

```diff
@@ -1,12 +1,12 @@
 {
 	"compilerOptions": {
-		"module": "esnext",
+		"module": "CommonJS",
```

Comment:
Ewww - do we have to?

Reply:
I got that from the example in https://docs.aws.amazon.com/lambda/latest/dg/lambda-typescript.html

But I don't think we need to follow that at all - the only reason why I reverted the `module` here is because this was causing runtime issues during development.

If I reset to `esnext` and I call my endpoint I get the error:
![Screenshot 2023-11-15 at 12 38 08 PM](https://github.com/thisdot/movies-api/assets/22064697/8c65fcb9-262b-400e-b46c-fdb8ae0a4f95)